assets/src/components/MousePosition.js

```diff
@@ -46,7 +46,8 @@ export default class MousePosition extends HTMLElement {
         return html`
             <div class="mouse-position">
                 <div class="editable-position ${['dm', 'dms', 'mgrs'].includes(this._displayUnit) ? 'hide' : ''}">
-                    <input type="number" step="any" placeholder="longitude" @input=${(event) => this._lonInput = parseFloat(event.target.value)} @keydown=${(event) => { if (event.key === 'Enter') { this._centerToCoords(); } }} .value=${isNaN(lon) ? 0 : lon}><input type="number" step="any" placeholder="latitude" @input=${(event) => this._latInput = parseFloat(event.target.value)} @keydown=${(event) => { if (event.key === 'Enter') { this._centerToCoords(); } }} .value=${isNaN(lat) ? 0 : lat}>
+                    <input type="number" step="any" placeholder="longitude" @input=${(event) => this._lonInput = parseFloat(event.target.value)} @keydown=${(event) => { if (event.key === 'Enter') { this._centerToCoords(); } }} .value=${isNaN(lon) ? 0 : lon}>
```

Comment:
Line break between `input`s might change the UI but I'm not sure so it has to be visually tested

Reply:
Ok, I have added two variables to split the line. Is that OK ?
Do you think it's ok in `assets/src/components/Snapping.js` or should I do the same ?